src/io/sendmsg_zc.rs

```diff
@@ -0,0 +1,58 @@
+use crate::io::SharedFd;
+use crate::runtime::driver::op::{Completable, CqeResult, MultiCQEFuture, Op, Updateable};
+use crate::runtime::CONTEXT;
+use std::io;
+
+pub(crate) struct SendMsgZc {
+    #[allow(dead_code)]
+    fd: SharedFd,
+
+    pub(crate) msghdr: libc::msghdr,
+
+    /// Hold the number of transmitted bytes
+    bytes: usize,
+}
+
+impl Op<SendMsgZc, MultiCQEFuture> {
+    pub(crate) fn sendmsg_zc(fd: &SharedFd, msghdr: &libc::msghdr) -> io::Result<Self> {
```

Comment:
I'm not sure about parameters that don't pass ownership of the underlying buffers to the operation. Could you pass a Vec\<IoSlice>?